.github/workflows/pythontests.yml

```
@@ -106,14 +106,11 @@ jobs:
       - name: Install dependencies
         run: |
           python -m pip install --upgrade pip
-          cd ../antsibull-core
-          pip install -e .
-          cd ../antsibull-docs
-          pip install -e .
-          pip install asynctest cryptography pytest "pytest-asyncio>=0.12"
+          pip install --use-pep517 -e . -e ../antsibull-core -e ../antsibull-docs \
```

```suggestion
          pip install --use-pep517 -e . -e ../antsibull-core \
```
You're installing antsibull-docs from `.` twice :)